contentdm_exporter/contentdm_record_exporter.py

```
@@ -128,44 +84,38 @@ def get_list_of_collection_aliases():
     """
 
     # GET collections
-    query_url_collections = "{main_url}dmGetCollectionList/json".format(
-        main_url=MAIN_URL
-    )
+    query_url_collections = f"{MAIN_URL}dmGetCollectionList/json"
 
     # Query CONTENTdm and return records; if failure, log problem.
     try:
         res = requests.get(query_url_collections)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)